examples/chip-tool/commands/common/Command.h

```diff
@@ -102,6 +102,12 @@ class Command
 
     virtual CHIP_ERROR Run(ChipDeviceController * dc, NodeId remoteId) = 0;
 
+    /*
+     * @brief
+     *  This is the exit status of the command
+     */
+    bool status = false;
```

Comment:
could you make this private with a getter? 
Public data members in classes are harder to reason about.

Reply:
Maybe also consider naming it:

```
bool mCommandExitStatus = false;
```

and then no doxygen comment may be needed (or if one is needed, I would post fix it to save some vertical space.